### PR TITLE
create new hardhat project

### PR DESCRIPTION
**Pull request by ezynda3**
create vscode workspace

fix compiler recommendations

add deployment scripts

This is a big commit but only because I copied the contracts over from the buidler package. This should be merged so I can contiue with the Factory contract.

### CODE REVIEW

File: packages/hardhat/hardhat.config.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (youngkidwarrior) [inline packages/hardhat/hardhat.config.ts:1]: I noticed this file is much smaller than the buidler file. This is bound to break some commands in the monorepo package.json. 

Can you comment out the broken commands and add TODOs

**Review by youngkidwarrior — CHANGES_REQUESTED** [review]

**Comment by ezynda3** [comment]
Should be fixed now

**Comment by youngkidwarrior** [comment]
@ezynda3 looks good!

There seems to be a conflict with your yarn.lock.

**Review by youngkidwarrior — DISMISSED** [review]

**Comment by youngkidwarrior** [comment]
@ezynda3 the factory contract looks good

Tasks left to be done:

* Remove deprecated buidler folder
* Add these commands for hardhat into `package.json` and finish the TODOs
```  "scripts": {
    "chain": "buidler node",
    "test": "buidler test --network buidlerevm",
    "compile": "buidler compile",
    "deploy": "buidler run scripts/deploy.js --show-stack-traces && buidler run scripts/publish.js",
    "watch": "node scripts/watch.js",
    "accounts": "buidler accounts",
    "balance": "buidler balance",
    "send": "buidler send"
  }```
  
  * Fix problems with Ownership on Facets

This should be contained in another PR that merges into this one before we merge to master. This will still count as 2 PRs in sourcecreds eyes

**Pull request merged**